homeassistant/components/overkiz/cover_entities/vertical_cover.py

```diff
@@ -100,13 +100,15 @@ async def async_set_cover_position(self, **kwargs: Any) -> None:
 
     async def async_open_cover(self, **kwargs: Any) -> None:
         """Open the cover."""
-        if command := self.executor.select_command(*COMMANDS_OPEN):
-            await self.executor.async_execute_command(command)
+        if not self.is_opened:
```

Comment:
And for closed state also, because in worst case a user can open manually a cover with remote and state will change in next polling. 

So, I realize there is no good solution, because the current implementation cover a desynchronization state beetwen HA states and real states. My fix cover an impossible transition, but we loose the advantage of current implementation.

life is made of compromises :sweat_smile: